templates/report.html.jinja

```diff
@@ -106,7 +108,89 @@
             </div>
         {% endif %}
     </div>
+    {% if has_rt_feed %}
+        <div class="flex-1 responsive-prose">
+            <h2>About the Realtime Completeness Charts</h2>
+            <ul>
+            <li>These charts show the percentage of Scheduled Trips with at least one Trip Update or Vehicle Position message assigned to that trip.</li>
+            <li>High percentages with flat lines are best and indicate large amounts of realtime information for customers.</li>
+            <li>Short drops may indicate schedule inaccuracies, missed runs (buses not on schedule), or hardware failures.</li>
+            </ul>
+        </div>
+    <div class="flex flex-col lg:flex-row gap-12 lg:gap-16 xl:gap-24 2xl:gap-36">
+        <div class="flex-1">
+            <div class="responsive-prose">
+                <h2>Trip Updates Completeness by&nbsp;Day</h2>
+            </div>
+            {% set tu_hours = gtfs_rt_completeness|list %}
+            <div class="hours-chart"
+                 data-dates="{{ getJsonForAttribute (tu_hours, 'service_date')|trim }}"
+                 data-hours="{{ getJsonForAttribute(tu_hours, 'percent_of_trips_with_TU_messages')|trim }}"
+                 data-color="#F6BF16"
+                 data-y-axis-label="% of trips with messages"
+                 data-tooltip-value-label="%"
+                 data-chart-type="area"
+                 data-chart-collabel="trip_update_completeness">
+            </div>
+        </div>
+        <div class="flex-1">
+            <div class="responsive-prose">
+                <h2>Vehicle Positions Completeness by&nbsp;Day</h2>
+            </div>
+            {% set vp_hours = gtfs_rt_completeness|list %}
+            <div class="hours-chart"
+                 data-dates="{{ getJsonForAttribute(vp_hours, 'service_date')|trim }}"
+                 data-hours="{{ getJsonForAttribute(vp_hours, 'percent_of_trips_with_VP_messages')|trim }}"
+                 data-color="#5B559C"
+                 data-y-axis-label="% of trips with messages"
+                 data-tooltip-value-label="percent"
+                 data-chart-type="area"
+                 data-chart-collabel="vehicle_position_completeness">
+            </div>
+        </div>
+    </div>
+        <div class="flex-1 responsive-prose">
+            <h2>About the Median Message Age&nbsp;Charts</h2>
+            <ul>
+            <li>These charts show the average median age of trip updates and vehicle position messages over time.</li>
```

Comment:
capitalize Trip Updates + Vehicle Positions for consistency w/ above?

Reply:
Done